tests/components/webostv/conftest.py

```diff
@@ -55,3 +56,28 @@ async def mock_state_update_callback():
         client.mock_state_update = AsyncMock(side_effect=mock_state_update_callback)
 
         yield client
+
+
+@pytest.fixture(name="client_entity_removed")
+def client_entity_removed_fixture(hass):
+    """Patch of client library, entity removed waiting for connect."""
+    with patch(
+        "homeassistant.components.webostv.WebOsClient", autospec=True
+    ) as mock_client_class:
+        client = mock_client_class.return_value
+        client.hello_info = {"deviceUUID": "some-fake-uuid"}
+        client.connected = False
+
+        def mock_is_conneted():
```

Comment:
mock_is_connected

Reply:
I think I need to change my keyboard, thanks, will fix soon 👍